packages/cardano-services/docker-compose.yml

```diff
@@ -9,7 +9,13 @@ x-logging: &logging
 
 services:
   cardano-node-ogmios:
-    image: cardanosolutions/cardano-node-ogmios:v${OGMIOS_VERSION:-5.6.0}_${CARDANO_NODE_VERSION:-1.35.5}-${NETWORK:-mainnet}
```

Comment:
Applying this change here rather than in `common.yml` makes the change not available for _local-testnet_ (e2e tests).
Is this as intend?

Applying this in `common.yml` rather than here will make _local-testnet_ less different from public networks, not only, will let the e2e tests to run against the custom Docker image (covering it).

I don't know if this is possible...

Reply:
Good catch @iccicci, I'll fix this before we merge